packages/embark/src/lib/modules/console/index.ts

```diff
@@ -102,8 +103,12 @@ class Console {
         let response = result;
         if (typeof result !== "string") {
           response = stringify(result, utils.jsonFunctionReplacer, 2);
+          this.logger.info(response);
```

Comment:
Does `stringify` handle escaping HTML that could produced from stringifying an object?

Reply:
It does not, but I'm pretty sure that this is the JSON printer, which we still want to be parsed as HTML.

The alternative is introducing plaintext / JSON response types that the cockpit could respond to, but that's a whole other bag of worms.

Reply:
Right, so when we `stringify` in this case, we return it in the response at the end. So I think maybe we should still escape it, no? For example,
```
        if (typeof result !== "string") {
          response = stringify(result, utils.jsonFunctionReplacer, 2);
          this.logger.info(response);
        } else {
          // Avoid HTML injection in the Cockpit
          this.logger.info(response);
        }
        response = escapeHtml(response);
        return res.send({ result: response });
```

Reply:
Isn't the JSON tree explorer rendered in the backend? Or is it the frontend that renders it?

Reply:
I think that's a frontend thing, but could be wrong.